src/libaktualizr/uptane/manifest.cc

```diff
@@ -0,0 +1,75 @@
+#include "manifest.h"
+
+#include "crypto/keymanager.h"
+
+namespace Uptane {
+
+Hash Manifest::installedImageHash() const {
+  // TODO: proper verification of the required fields
```

Comment:
What does "proper" mean in each of these cases?

Reply:
Handling the case when the required key(s) are not present in the json.